lms/services/vitalsource/_client.py

```diff
@@ -103,10 +120,12 @@ class Meta:
             unknown = EXCLUDE
 
         title = fields.Str(required=True)
+        """Title of the chapter."""
+
         cfi = fields.Str(required=True)
-        page = fields.Str(required=True)
+        """A reference to the location within the book."""
 
-        url = fields.Str(required=False)
-        """vitalsource:// like url identifying the book and chapter"""
+        page = fields.Str(required=True)
+        """The start page of the chapter."""
```

Comment:
URL has been removed from the schema. See: https://github.com/hypothesis/lms/pull/4259/commits/8a538ba7a419c7b4d503a2eff957399bdcd3131f for details